readers/cassandra/messages_test.go

```diff
@@ -147,7 +147,10 @@ func TestReadAll(t *testing.T) {
 	}
 
 	for desc, tc := range cases {
-		result := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		result, err := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		if err != nil {
+			assert.Equal(t, nil, err, fmt.Sprintf("%s: expected no error got %s", desc, err.Error()))
+		}
```

Comment:
Don't use `if err != nil {`. Use `assert.Nil` instead of this `if` and `assert.Equal`.

Reply:
Thanks!